detox/test/e2e/06.device.test.js

```diff
@@ -54,4 +54,41 @@ describe('Device', () => {
     await device.shake();
     await expect(element(by.text('Shaken, not stirred'))).toBeVisible();
   });
+
+  describe('device orientation', () => {
+    beforeEach(async() => {
+      await device.reloadReactNative();
+      await element(by.text('Orientation')).tap();
+
+      // Check if the element which input we will test actually exists
+      await expect(element(by.id('currentOrientation'))).toExist();
+    });
+
+    it('OrientationLandscape', async () => {
+      await device.setOrientation('landscape');
+
+      await expect(element(by.id('currentOrientation'))).toHaveText('Landscape');
+    });
+
+    it('OrientationPortrait', async() => {
+      // As default is portrait we need to set it otherwise
+      await device.setOrientation('landscape');
+      await device.setOrientation('portrait');
+
+      await expect(element(by.id('currentOrientation'))).toHaveText('Portrait');
+    });
+  });
+
+  describe('device button', () => {
```

Comment:
I see why iOS builds fail, add `:android:` to the test name to make sure iOS tests skip it.
Also, change the test name to something more descriptive if possible.

Reply:
done